test/helpers/files.spec.ts

```diff
@@ -63,7 +59,7 @@ const REWRITES: Rewrites = [
 ]
 
 describe('files utility functions', () => {
-  test('middleware tester matches correct paths', () => {
+  it('middleware tester matches correct paths', () => {
```

Comment:
Is `it` what we use everywhere else? I can't remember between all the projects we have. 😅

Reply:
I've seen `it` used in e.g. Gatsby plugin, too